### ISSUE TITLE
Accessibility issue on the carbon search box on https://ibmdotcom-react.netlify.com/iframe.html?id=dotcom-shell--default

### ISSUE THREAD

**Issue by alainmachu** [carbon-design-system/ibm-dotcom-library#588]
-->

**## Detailed description**

> Describe in detail the issue you're having.
On https://ibmdotcom-react.netlify.com/iframe.html?id=dotcom-shell--default, expand the search box and type java
It triggers a lot of suggestions
If we use the down arrow on the keyboard to go through the suggestions, we see them in the search box, as we go down, but they are not highlighted in the list below the search box

Best practice for it would be, that as we go through the list with the down or up key, and a suggestion is showing in the search box, the same suggestion should be highlighted in the list below.

> Is this a feature request (new component, new icon), a bug, or a general
> issue? it's an accessibility issue, best practice is that in the scenario above, the suggestions are highlighted as we go through them with the down or up keyword.

> Is this issue related to a specific component? Carbon

> What did you expect to happen? What happened instead? What would you like to
> see changed? See above

> What browser are you working in? Firefox and Chrome

**## Steps to reproduce the issue**

1. Step one: load https://ibmdotcom-react.netlify.com/iframe.html?id=dotcom-shell--default
2. Step two: enter java in the search box
3. Step three: you will see suggestion list below the search box
4. Use down key in the board to go through the list
5. the first suggestion is javascript
6. While using the the down key, javascript shows in the search box in place of java, but it is NOT highlighted in the list
7. And so on for all the keywords
